ludwig/features/binary_feature.py

```diff
@@ -252,8 +239,6 @@ def create_preproc_module(metadata: Dict[str, Any]) -> torch.nn.Module:
 
 
 class BinaryOutputFeature(BinaryFeatureMixin, OutputFeature):
-    metric_functions = {LOSS: None, ACCURACY: None, ROC_AUC: None}
-
```

Comment:
Seems like you are missing a call to `BinaryOutputFeatureConfig.get_output_metric_functions()` here right?

Reply:
fixed